noxfile.py

```diff
@@ -46,7 +39,7 @@ def run_pybamm_requires(session):
     """Download, compile, and install the build-time requirements for Linux and macOS: the SuiteSparse and SUNDIALS libraries."""  # noqa: E501
     set_environment_variables(PYBAMM_ENV, session=session)
     if sys.platform != "win32":
-        session.run_always("pip", "install", "wget", "cmake")
```

Comment:
Using `session.run_always()` as an alias to `session.install` was only needed for local development for cases where `pip` did not display an output. It was and is fine to use it in CI